include/ncengine/graphics/SkeletalAnimationTypes.h

```diff
@@ -9,18 +9,24 @@
 
 namespace nc::graphics::anim
 {
+static constexpr uint32_t NullState = UINT32_MAX;
```

Comment:
Fixed static. I prefer omitting the "Index" because from the client code it's a little nicer to see: `.exitTo = anim::NullState` and I've been following a convention of referring to state indices as "state" in the client code, as that's the relevant info to them